### ISSUE TITLE
Checkbox needs focus styles

### ISSUE THREAD

**Issue by jes708** [priceline/design-system#343]
When tabbing over design-system™ checkboxes, there is no focus styling to show the user which checkbox is currently selected.

**Comment by James300** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Checkbox needs focus styles](https://trello.com/c/n7C776o9/18-checkbox-needs-focus-styles)
